docs/getting-started/user-guides/using-paymetheus.md

```diff
@@ -0,0 +1,151 @@
+
+![Overview tab](/img/Paymetheus-accounts.png)  
+
+---
+
+
+## **Scripts** ##
+Currently only used for PoS mining in a pool. If you sign up for a pool, 
+you will be asked to import a script. This is where you will add it.
+See the Purchase Tickets section below for more information.  
+
+![Scripts tab](/img/Paymetheus-import-script.png)  
+
+---
+
+
+## **Create Transaction** ##
+This tab is how you send funds to another address. Simply copy the receivers
+address into the text box and type in the amount of Decred you wish to send.
+The estimated fee will be listed as well. You can click the '+' button to 
+send Decred to multiple accounts in the one transaction if you wish.  
+
+![Create transaction tab](/img/Paymetheus-send.png)  
+
+---
+
+
+## ** Purchase Tickets tab** ##
+
+Stake pools are also usable with Paymetheus by using its manual ticket
+purchasing features.  
+
+> To join the pool, provide a public key address which can be used to generate a
+> 1-of-2 multisignature script. The multisignature script will be generated by
+> the pool and returned to you along with a P2SH address to give voting rights to.  
+
+Don't worry if you didn't understand that quote. What it means is that you create
+an address that can be accessed by two wallets. Only one wallet needs to be available
+to use the address. This means that the pool can vote on your behalf and you can vote
+using your own wallet if the pool stops working.  
+
+It DOES NOT give the pool access to your funds. All you are doing is granting voting
+rights to the wallet. This is why you provide a ticket address so the network knows
+where to send your funds to after the ticket votes/expires. The pool does not touch
+your funds.  
+
+It is recommended that you generate a new account when joining a stake pool. This is
+because accounts are hardened so in the case of a total stake pool
+failure/shutdown, it would be safe to give the private key to another stake pool
+as long as that account only does voting and nothing else.  
+
+Official stake pools are [listed here](/mining/proof-of-stake#sign-up-for-a-stake-pool).
+All stake pools run the same basic code, but may differ in the amount of redundancy available.
+More redundancy equals less chance of missed votes (although all pools will have missed votes
+as many missed votes are caused by PoW miners). In order to ensure one pool does't become too large,
+it is recommended that you join a smaller pool. This further minimizes the (VERY) small chance that a pool
+owner could exert influence over the way their pool votes if it's too large. Decred already contains checks
+for this, but by spreading tickets around pools, it makes the network even less decentralized.
+
+![Creating voting account](/img/Paymetheus-create-voting-account.png)  
+
+To generate a public key address, use the "Request Payment" view to generate a
+new address for the voting account.  The pubkey address (below the shorter P2PKH
+address) will be given to the pool.
```

Comment:
"will be given" should be "must be given", these are instructions the user must currently follow before tighter stakepool integration is implemented.